### PR TITLE
Update for Swift 5 / RxSwift 5

### PR DESCRIPTION
**Pull request by mwoollard**
Have updated for with Swift 5 / RxSwift 5, appreciate if this can be merged and published as using in project in process of updating to Swift 5 and this is the one pod blocking this:-)

Thanks!

### CODE REVIEW

**Comment by freak4pc** [comment]
Thanks @mwoollard !

I think it could be nice to also change the generic names of this framework (e.g. O to something more obvious), but it doesn't have to be part of this PR... lets get this rolling first :) 

Can you fix the CI? You might need to disable the cache step, run a build, and then put the cache step back. CircleCI is a bit meh in that regard.

File: Example/Podfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-platform :"ios", "8.0"
+platform :"ios", "10.0"
```
Comment (freak4pc) [inline Example/Podfile:1]: Any reason for the bump? RxSwift still supports iOS 8 and up ... so not sure why the bump is needed

**Comment by freak4pc** [comment]
Hey @mwoollard - I'm sorry but I just realized there was already a long-standing PR open with these changed (#45). I've merged that one and will close this instead. Thank you, regardless!

File: Example/Podfile (hunk 1 above)
Comment (mwoollard) [reply Example/Podfile:1]: My mistake!

**Comment by mwoollard** [comment]
No problem - as long as we get the update published :-)

**Pull request closed without merging**